WordPress/src/test/java/org/wordpress/android/ui/sitecreation/domains/NewSiteCreationDomainsViewModelTest.kt

```diff
@@ -140,10 +142,11 @@ class NewSiteCreationDomainsViewModelTest {
      */
     @Test
     fun verifyNonEmptyUpdateQueryInitialUiState() = testWithSuccessResponse {
+        viewModel.start(null, SEGMENT_ID)
```

Comment:
I am a bit sad about having to add the `start` calls to these tests, but I couldn't think of a great way to handle it without it.